src/goto-checker/solver_factory.cpp

```diff
@@ -331,7 +331,8 @@ solver_factoryt::get_string_refinement()
 
 std::unique_ptr<std::ofstream> open_outfile_and_check(
   const std::string &filename,
-  message_handlert &message_handler)
+  message_handlert &message_handler,
+  const std::string &arg_name)
```

Comment:
Random place for comment: the commit message needs to be reviewed (spurious "---------------------", missing words after "that is being").